.gitignore

```diff
@@ -8,7 +8,6 @@
 
 # production
 /build
-/dist
```

Comment:
Please put restore `/dist` in `.gitignore`

Reply:
Thanks for the correction